app/models/requests/service_eligibility/ill.rb

```diff
@@ -29,7 +30,15 @@ def user_eligible?
             user.cas_provider? || user.alma_provider?
           end
 
-          attr_reader :requestable, :user, :any_loanable
+          def patron_eligible?
```

Comment:
Should this be [def patron_group_eligible](https://github.com/pulibrary/orangelight/blob/78519f59dfcc6e32ad1eb16999a32f70c74c9620/app/models/requests/service_eligibility/abstract_on_shelf.rb#L42)?

Reply:
Fixed